### PR TITLE
Drop cycleway=no property

### PR DESCRIPTION
**Pull request by rmarianski**
Connects to https://github.com/mapzen/vector-datasource/issues/774

This only addresses the part of the issue related to cycleway's.

@nvkelso could you review please?

### CODE REVIEW

**Comment by nvkelso** [comment]
Word.

File: yaml/roads.yaml [hunk 1]
```diff
@@ -34,7 +34,9 @@ global:
       bicycle_network: {expr: "mz_cycling_network(tags, osm_id)"}
       oneway: {col: oneway}
       oneway_bicycle: {col: "tags->oneway:bicycle"}
-      cycleway: {col: tags->cycleway}
+      cycleway:
+        expr: |
+          CASE WHEN tags->'cycleway' <> 'no' THEN tags->'cycleway' END
       cycleway_left: {col: "tags->cycleway:left"}
```
Comment (nvkelso) [inline yaml/roads.yaml]: Each of the variants `cycleway_left`, `cycleway_right`, `cycleway_both`, needs the `no` logic too.

Comment (nvkelso) [inline yaml/roads.yaml]: `none` looks common to as a synonym for `no`.

**Comment by rmarianski** [comment]
Updated in 80f256e. Look ok now?

Also, the numbers for [cycleway:both](http://taginfo.openstreetmap.org/keys/cycleway:both#values) look small. Should we just drop that completely?

**Comment by nvkelso** [comment]
LGTM

**Pull request merged**